crates/sovereign-sdk/rollup-interface/src/state_machine/zk/mod.rs

```diff
@@ -173,8 +173,10 @@ pub struct BatchProofCircuitInput<'txs, StateRoot, Witness, Da: DaSpec, Tx: Clon
     /// DA block headers the soft confirmations was constructed on.
     pub da_block_headers_of_soft_confirmations: VecDeque<Vec<Da::BlockHeader>>,
     /// Sequencer soft confirmation public key.
+    /// **DO NOT USE THIS FIELD IN POST FORK1 GUEST**
```

Comment:
Use this:
```suggestion
    /// **DO NOT USE THIS FIELD IN POST FORK1 GUEST**
    #[deprecated]
```

Reply:
We actually shouldnt because we are still using it until we are officially past Fork1.

But still great suggestion because this allowed me to see one part I missed: https://github.com/chainwayxyz/citrea/pull/1613/commits/b9a02174a2e4cba129ffff9b339b3377381f132c